lib/Slackbot_worker.js

```diff
@@ -27,11 +27,11 @@ module.exports = function(botkit, config) {
             botkit.debug('CANNOT SEND WEBHOOK!!');
 
             return cb && cb('No webhook url specified');
-        }
+  }
 
         request.post(bot.config.incoming_webhook.url, function(err, res, body) {
-            if (err) {
-                botkit.debug('WEBHOOK ERROR',err);
+          if (err) {
```

Comment:
Aha! I'll fix those. How are you running `jscs`? When I run `jscs ./lib` I get hundreds of errors. Can I suggest adding `"pretest": "jscs ./lib"` to `npm scripts` so others don't make the mistake I made?